src/start.sh

```diff
@@ -75,10 +75,9 @@ start() {
         sleep 0.5
     done
 
-    #  Wait until the FTL log contains the "FTL started" message before continuing
-    while ! grep -q '########## FTL started' /var/log/pihole/FTL.log; do
-        sleep 0.5
-    done
+    #  Wait until the FTL log contains the "FTL started" message before continuing, timeout after 10 seconds
+    logsize_before=$(stat -c%s /var/log/pihole/FTL.log)
```

Comment:
Do we need to get this?

Reading the comments in the FTL PR, we can just pass in 0 and it will start reading from the end of the file. I feel like not much time will pass between lines 79 and 80 it worth populating the variable?

Reply:
Probably we don't need this.